pkg/assert/pod.go

```diff
@@ -129,3 +129,16 @@ func PodStatusRunning(namespace, label string) {
 		g.Expect(err).NotTo(HaveOccurred(), err)
 	}, "30s", "5s").Should(Succeed())
 }
+
+// ValidateIntraNSPodConnectivity ensures that one pod, the "server", can be reached from another, the "client"
+// within the same namespace
+func ValidateIntraNSPodConnectivity(namespace, clientPodName, serverPodIP, expectedResult string) {
```

Comment:
is this args ever going to be used? 

because you created a function that its called 3 times with exact same parameters values

shouldnt we create functions with parameters so that its "configurable " to be used with diff values? 

here u could in the caller directly call the validateHost passing your stuff like
```
	execErr := assert.ValidateOnHost(execCommand, "Hostname: server")
	if execErr != nil {
		return
	}
```

Reply:
I created this as its own function because the `serverPodIP` changes a few times, so I found myself writing the same function a few times for slightly different arg values.

Reply:
so you think worthwhile creating that rather then just 
```
//validate Intra NSPodConnectivity
 	execCommand := fmt.Sprintf(
		"kubectl exec -it -n %s pod/%s --kubeconfig=%s -- wget -O - http://%s",
		"test-k3s-issue-10053", "client", shared.KubeConfigFile, serverPodIP)

	execErr := assert.ValidateOnHost(execCommand, "Hostname: server")
	if execErr != nil {
		return
	}
```
just asking, not saying its better or anything

Reply:
yeah I like having it separate so as to duplicate code less